packages/daemon/src/service/log-manager.ts

```diff
@@ -57,6 +54,17 @@ class LogPassthrough extends Transform {
     callback();
   }
 
+  /**
+   * cover Transform.write, otherwise if no `data` event listener,
+   * the callback `_transform` will not be called, but we need to save the log to file.
+   * @param args args for stream.write
```

Comment:
This description is incorrect, rest parameters should not be supported by typedoc.